src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -231,6 +223,29 @@ protected void shutdown() throws InterruptedException {
         certificateManager.stopMonitors();
     }
 
+    @Override
+    public void postInject() {
+        super.postInject();
+        try {
```

Comment:
TODO: We can consider migrating certificate APIs in a follow up PR, which will require a MQTT bridge CDA dependency bump as well